scene/3d/physics_body.h

```diff
@@ -114,7 +114,7 @@ class RigidBody : public PhysicsBody {
 		MODE_KINEMATIC,
 	};
 
-private:
+protected:
```

Comment:
Why change that? There's two `protected:` blocks now.

Reply:
woops, didnt notice that. fixed it